ncl/ncl_entries/general_applied_math.ipynb

```diff
@@ -0,0 +1,810 @@
+    "### Grab and Go"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 13,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "[ 1  3  6 10 15]\n"
+     ]
+    }
+   ],
+   "source": [
+    "import numpy as np\n",
+    "\n",
+    "input_values = [1, 2, 3, 4, 5]\n",
+    "\n",
+    "cumsum_output = np.cumsum(input_values, axis=0)\n",
+    "print(cumsum_output)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## decimalPlaces\n",
+    "\n",
+    "NCL's `decimalPlaces` function truncates or rounds to the number of decimal places specified"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Grab and Go"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 14,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "3.142\n"
+     ]
+    }
+   ],
+   "source": [
+    "# round: Single Value\n",
+    "input_value = 3.1415926\n",
+    "\n",
+    "round_output = round(input_value, 3)\n",
+    "print(round_output)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 15,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "[-10.354   3.142 123.457]\n"
+     ]
+    }
+   ],
+   "source": [
+    "# round: Array\n",
+    "import numpy as np\n",
+    "\n",
+    "input_values = [-10.353535, 3.1415926, 123.4567]\n",
+    "\n",
+    "round_output = np.round(input_values, 3)\n",
+    "print(round_output)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 16,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "3.141\n"
+     ]
+    }
+   ],
+   "source": [
+    "# truncate: Single Value\n",
+    "import math\n",
+    "\n",
+    "input_value = 3.1415926\n",
+    "\n",
+    "decimal_values = str(input_value).split(\".\")\n",
+    "truncate_decimal = decimal_values[1][:3]\n",
+    "truncate_output = float(decimal_values[0] + \".\" + truncate_decimal)\n",
+    "print(truncate_output)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Differences from NCL\n",
+    "\n",
+    "NCL distinguishes between rounding a decimal value and truncating it as part of the `decimalPlaces` function\n",
+    "\n",
+    "```\n",
+    "decimalPlaces(3.145, 2, True)  # True = rounds\n",
+    ">> 3.15\n",
+    "decimalPlaces(3.145, 2, False) # False = truncates\n",
+    ">> 3.14\n",
+    "```\n",
+    "\n",
+    "Python includes multiple functions for rounding a value (`math.ceil`, `math.floor`, `round`), but does not typically handle truncating a value. To truncate a decimal value, the decimal needs to be converted to a string first.\n",
+    "\n",
+    "The `round` function returns the valuerounded to the ndigit after the decimal point\n",
```

Comment:
```suggestion
    "The `round` function returns the value rounded to ndigits after the decimal point\n",
```

Reply:
Yes, the additional numpy work is in v2, but I expanded on this to detail the kind of rounding that is happening here with some disclaimers